portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,9 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <img src="/images/PROFILE_PIC.JPG">
+      <h1>Michaela Murray's Portfolio</h1>
+      <p id="intro"> Welcome! My name is Michaela. I am currently a rising junior at Stanford University!</p>
```

Comment:
nit - Although I can't find specific guidance in the [html style guide](https://google.github.io/styleguide/htmlcssguide.html) - all of their examples don't have whitespace between the start-tag and the content. I would drop the space.

Reply:
Thanks for the input! I looked through my current HTML file and made sure all spaces were dropped.